tests/mocks.py

```diff
@@ -131,10 +168,69 @@
     }
 }
 
+# One version of products named 'cos', 'sat' and 'cpe' that has valid YAML but not matching schema
+# - 'cos' product with invalid component manifests
+# - 'sat' product with invalid component docker
+# - 'cpe' product with invalid component s3
+MOCK_INVALID_PRODUCT_DATA = {
+    'cos': {
+        '2.1': {
+            'component_versions': {
+                'manifests': 'should be an array'
```

Comment:
Should also add a check like this for `s3`, since it's also supposed to be an array.
Also, add a check where `manifests` is an array that has at least one non-string value, since the schema requires it to accept only string values.